configure.ac

```diff
@@ -603,13 +603,15 @@ if test "$enable_cloud" != "no"; then
     else
         enable_aclk=$aclk_required
```

Comment:
I was trying to trace `aclk_required` and I can't see any case that leads to it having different values to `enable_cloud`. Some parts of the code use one, some the other but I don't see them diverging whatsoever. Am I missing something?

Reply:
I don't remember that part of the logic, it was from when we had the --cloud-testing flag. The --require-aclk was added to force the build to break if the deps were not met. That could be the same as enable_cloud?

Reply:
@stelfrag @underhood If anyone remembers what's the deal here please chip in. We could merge the 2 and just have one variable, whatever name makes more sense. It would be much easier to read the code.

Reply:
Yes. When the flag was changed in past it made having 2 vars obsolete.